<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QDialog" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Infomation</string>
  </property>
  <widget class="QLabel" name="label_userID">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);
border: none;</string>
   </property>
   <property name="text">
    <string>UserNamej</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sign">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);
border: none;</string>
   </property>
   <property name="text">
    <string>你所热爱的，就是你的生活。</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{background:rgba(0,0,0,0);border:none;}
QComboBox::drop-down{border-style:none;}
QComboBox::item{background:rgb(0,0,0);}</string>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgba(255,255,255,128);
	border-radius: 5px;
}
QPushButton::hover
{
	background: rgba(255,255,255,192);
	border-radius: 5px;
}
QPushButton::pressed
{
	background: rgba(255,255,255,255);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgba(255,255,255,128);
	border-radius: 5px;
}
QPushButton::hover
{
	background: rgba(255,255,255,192);
	border-radius: 5px;
}
QPushButton::pressed
{
	background: rgba(255,255,255,255);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_age">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0,0,0,0);
border: none;</string>
   </property>
   <property name="text">
    <string>18</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_gender">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>191</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Medium ITC</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{background:rgba(0,0,0,0);border:none;color:black;}
QComboBox::drop-down{border-style:none;}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Unknown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Resources/LoginBG.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cover">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>comboBox_icon</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>label_userID</zorder>
  <zorder>lineEdit_sign</zorder>
  <zorder>pushButton_edit</zorder>
  <zorder>pushButton_submit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_age</zorder>
  <zorder>comboBox_gender</zorder>
  <zorder>label_cover</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
